connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/webhook/VerifiableWebhook.java

```diff
@@ -32,12 +33,17 @@ public WebhookHttpVerificationResult(Object body) {
       this(body, null, 200);
     }
 
-    public WebhookHttpVerificationResult(Object body, int statusCode) {
+    public WebhookHttpVerificationResult(Object body, Integer statusCode) {
       this(body, null, statusCode);
     }
 
     public WebhookHttpVerificationResult(Object body, Map<String, String> headers) {
       this(body, headers, 200);
     }
+
+    @Override
+    public Integer statusCode() {
+      return statusCode == null || statusCode < 100 ? 200 : statusCode;
```

Comment:
@igpetrov Records supports custom constructors for these use-cased: https://www.baeldung.com/java-records-custom-constructor#how-to-make-a-custom-constructor-for-a-java-record

Reply:
There you can put 200 as the default if you like

Reply:
The benefit is that you construct a valid object that has the right values. Similar to kotlin data classes.